.github/workflows/canary-ap-northeast-2.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           envkey_REDIS_LOCAL_TTL_FACTOR: ${{ secrets.REDIS_LOCAL_TTL_FACTOR }}
           envkey_BLOCKED_ADDRESSES_URL: ${{ secrets.BLOCKED_ADDRESSES_URL }}
           envkey_PHD_BASE_URL: ${{ secrets.PHD_BASE_URL }}
-          envkey_PHD_API_KEY: ${{ secrets.PHD_API_KEY }}
+          envkey_PHD_API_KEY_SERVICE_PRODUCTION: ${{ secrets.PHD_API_KEY }}
```

Comment:
Isn't the name of the secret itself going to change now?

Reply:
good catch! you're totally right